### PR TITLE
feat: Added a system test for prediction cpr.

### PR DESCRIPTION
**Pull request by abcdefgs0324**

### CODE REVIEW

File: tests/system/aiplatform/test_prediction_cpr.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Google LLC
```
Comment (samthrasher) [inline tests/system/aiplatform/test_prediction_cpr.py:3]: 2022?

File: tests/system/aiplatform/test_resources/cpr_user_code/predictor.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+from typing import Any
+
+from google.cloud import storage
+from google.cloud.aiplatform.prediction.predictor import Predictor
+
+
+class SklearnPredictor(Predictor):
+    """Interface for Predictor class that users would be implementing."""
+
+    def __init__(self):
```
Comment (samthrasher) [inline tests/system/aiplatform/test_resources/cpr_user_code/predictor.py:29]: is it necessary to override `__init__` if it has no effect?

**Review by samthrasher — APPROVED** [review]
client code looks great!

**Review by samthrasher — CHANGES_REQUESTED** [review]
please fix build before submitting

File: tests/system/aiplatform/test_resources/cpr_user_code/predictor.py (hunk 2 above)
Comment (abcdefgs0324) [reply tests/system/aiplatform/test_resources/cpr_user_code/predictor.py:29]: `__init__` in predictor template is marked as `NotImplementedError` by default.

**Review by kthytang — APPROVED** [review]

**Pull request merged**